### PR TITLE
Load tracing information from environment

### LINKED ISSUE #1906: Continue trace from ENV vars
It should be able to continue a trace with trace information that was given to the python process over environment variables. 

See this RFC for the spec: https://github.com/getsentry/rfcs/blob/main/text/0071-continue-trace-over-process-boundaries.md

### PR DESCRIPTION
**Pull request by antonpirker**
It should be able to continue a trace with trace information that was given to the python process over environment variables.

See this RFC for the spec: 
https://github.com/getsentry/rfcs/blob/main/text/0071-continue-trace-over-process-boundaries.md

FIxes #1906

### CODE REVIEW

**Comment by antonpirker** [comment]
In our daily today it was decided to put this on hold for now.

**Comment by stephanie-anderson** [comment]
> In our daily today it was decided to put this on hold for now.

Was a misunderstanding, please go forward with this as it is already been decided to do so in an RFC and Sentry itself would need this (@mitsuhiko)

File: sentry_sdk/scope.py [hunk 1]
```diff
@@ -121,7 +122,49 @@ def __init__(self):
...
+            "NO",
+            "n",
+            "0",
+            "off",
+            "Off",
+            "OFF",
+        ]
+
+        use_environment = (
+            False if os.environ.get("SENTRY_USE_ENVIRONMENT") in false_values else True
```
Comment (stephanie-anderson) [inline sentry_sdk/scope.py:152]: Could we have `os.environ.get("SENTRY_USE_ENVIRONMENT")` in a variable and do a lowercase transformation on it? Then we would not have to maintain a list of possible lower/uppercase combinations of `false`, `no`, `off` and so on (see `false_values` array)

Comment (antonpirker) [reply sentry_sdk/scope.py:152]: good point. yes we can!

**Comment by antonpirker** [comment]
I messed this branch up. Will create a new, clean one.

**Pull request closed without merging**